extras/gen_release_candidate_changes.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
```

Comment:
We need a way for people doing releases to know that this exists (in case someone else needs to do them)

I think we should guide them to use this in our release guide.
What do you think?

Reply:
That would be great.